salt/modules/dockerng.py

```diff
@@ -102,6 +102,15 @@
 Both methods can be combined; any registry configured under
 ``docker-registries`` or ``*-docker-registries`` will be detected.
 
+Configuration Options
+---------------------
+
+The following options can be set in the :ref:`minion config
+<configuration-salt-minion>`:
+
+- ``docker.url``: URL to the docker service (default: local socket).
+- ``docker.version``: API version to use (default: currently 1.4 API).
+- ``docker.exec_driver``: Execution driver to use (default: docker-exec).
 
```

Comment:
This is incorrect. Refer to [this section](https://github.com/saltstack/salt/blob/1d00cee/salt/modules/dockerng.py#L177-L187), the value of this option will not ever default to ``docker-exec``, it will vary based on the container. For most containers it will be ``nsenter``, but for older containers it will be ``lxc-attach``. It's probably best to just link down to the section I highlighted to explain how this parameter is handled.